### PR TITLE
honto.jp

### PR DESCRIPTION
**Pull request by SKEIDs**
#105153

### CODE REVIEW

File: MobileFilter/sections/specific_web.txt [hunk 1]
```diff
@@ -27,6 +27,7 @@ theinventory.com,avclub.com,clickhole.com,deadspin.com,gizmodo.com,jalopnik.com,
 ! ##.ads
 cartoonpornvideos.com,m.economictimes.com,beethovenmaster.com,m.mistrzowie.org,m.ura.news,siirtpress.com,up.ymgt.xyz,google.de,www-bento-de.cdn.ampproject.org,m-economictimes-com.cdn.ampproject.org,m.winfuture.de##.ads
 !
+honto.jp##.momRegBanner
```
Comment (Yuki2718) [inline MobileFilter/sections/specific_web.txt:30]: Move this rule to Annoyances > self-promo.

File: SocialFilter/sections/specific.txt [hunk 2]
```diff
@@ -48,6 +48,7 @@ sedaily.com,canshop.jp,koreatimes.co.kr,m.chosun.com,sports.donga.com##.sns_area
 ! ##.content__post--meta-social
 otomobilgunluklerim.com,tilki.net,ekosayfa.com,bilisimogretmeni.com,azbuz.org,siberkalem.com,teknobird.com,buneymis.net,teknoxhaber.com,fosforludusunceler.com,kirtkirtla.com,oyuncukuru.org,denizcilikbilgileri.com,hknkr.com,kadioglublog.com##.content__post--meta-social
 !
+honto.jp##.stSns01
```
Comment (Yuki2718) [inline SocialFilter/sections/specific.txt:51]: Is this typo? On my end it's `##.stSnsList01`.

**Review by Yuki2718 — CHANGES_REQUESTED** [review]

File: SocialFilter/sections/specific.txt (hunk 2 above)
Comment (SKEIDs) [reply SocialFilter/sections/specific.txt:51]: Sorry.
I had only added the rules for PC.
I forgot the rules for mobile.
`##.stSnsList01` is for mobile rule.

File: MobileFilter/sections/specific_web.txt (hunk 1 above)
Comment (SKEIDs) [reply MobileFilter/sections/specific_web.txt]: done

File: SocialFilter/sections/specific.txt (hunk 2 above)
Comment (SKEIDs) [reply SocialFilter/sections/specific.txt:51]: <details><summary>mobile</summary>


</details><br/>


<details><summary>PC</summary>


</details><br/>

Comment (Yuki2718) [reply SocialFilter/sections/specific.txt:51]: You're right.

**Review by Yuki2718 — APPROVED** [review]

**Review by Divine-Frog — APPROVED** [review]

**Pull request merged**